modules/gapi/include/opencv2/gapi/own/cvdefs.hpp

```diff
@@ -139,15 +138,23 @@ enum InterpolationFlags{
     INTER_LINEAR_EXACT   = 5,
     INTER_MAX            = 7,
 };
+
+// struct GMatDesc; // another way - without include gmat.hpp
+
 // replacement of cv's structures:
 namespace gapi { namespace own {
 class Mat;
+// GMatDesc descr_of(const Mat &mat); // another way: error-> conflicts with a previous declaration in gmat.hpp 
```

Comment:
Another way - we don't include gmat.hpp. We put a function prototype. 

`cv::gapi::own::descr_of(const gapi::own::Mat &)`